spec/features/cards_check_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+feature "Check cards" do
+  context "cards are available for training" do
+    
+    let!(:card1) { create(:card) }
+    let!(:card2) { create(:card, original_text: "someword", translated_text: "you") }
+    
+    before do
+      visit root_path
+    end
+
+    scenario "opens successfully" do
+      expect(page.status_code).to be 200
+    end
+
+    scenario "enter valid translation" do
+      fill_in 'card_translated_text', with: "you"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.